src/renderer/src/routes/pages/Send/SendAmount.tsx

```diff
@@ -65,22 +66,29 @@ export const SendAmount = ({
               onSelectAmount: onSelectAmount,
             },
           })}
-        >
-          {selectedAmount > 0 ? (
-            <span className="mr-3 text-gray-100">{selectedAmount}</span>
-          ) : (
-            <span
-              className={StyleHelper.mergeStyles('mr-3', {
-                'text-neon': active,
-                'text-gray-100': !active,
-              })}
-            >
-              {t('inputAmount')}
-            </span>
-          )}
-
-          <TbChevronRight className="w-5 h-5 text-gray-300" />
-        </button>
+          clickableProps={{ className: selectedToken ? 'hover:bg-gray-300/15 hover:rounded pr-1' : ' pr-1' }}
+          variant="text"
+          colorSchema={active ? 'neon' : 'white'}
+          label=""
+          labelComponent={
```

Comment:
You can use the label props instead
```suggestion
        label={selectedAmount > 0 ? selectedAmount.toString() : t('inputAmount')}
```